README.md

```diff
@@ -219,35 +203,67 @@ Mutation output is grouped by selection groups. Each group contains three sectio
     end
    -----------------------
    ```
+   
+Concurrency
+-----------
 
-Presentations
--------------
+By default, mutant will test mutations in parallel by running up to one process for each core on your system. You can control the number of processes created using the --jobs argument.
+ 
+Mutant forks a new process for each mutation to be tested to prevent side affects in your specs, and the lack of thread safety in rspec, from impacting the results.
 
-There are some presentations about mutant in the wild:
+If the code under test relies on a database, you may experience problems when running mutant because of conflicting data in the database. For example, if you have a test like this:
+```
+m = MyModel.create!(...)
+expect(MyModel.first.name).to eq(m.name)
+```
+It might fail if some other test wrote a record to the MyModel table at the same time as this test was executed. (It would find the MyModel record created by the other test.) Most of these issues can be fixed by writing more specific tests. Here is a concurrent safe version of the same test:
+```
+m = MyModel.create!(...)
+expect(MyModel.find_by_id(m.id).name).to eq(m.name)
+```
+You may also try wrapping your test runs in transactions, rather than fixing your tests.
```

Comment:
I'd remove the

> rather than fixing your tests

As I think you should do "both", fixing your tests to not be sequence dependent, and isolating them via system transactions.